uclapi.openapi.json

```diff
@@ -0,0 +1,4256 @@
+          },
+          "device_type" : {
+            "type" : "string",
+            "example" : "Desk",
+            "description": "Type of seat. Usually this is Desk."
+          },
+          "host_address" : {
+            "type" : "string",
+            "example" : "584",
+            "description": "Address of host."
+          },
+          "building_name" : {
+            "type" : "string",
+            "example" : "IOE (20 B-Way)",
+            "description": "A short but human-readable name of the building that the sensor is in."
+          },
+          "room_description" : {
+            "type" : "string",
+            "example" : "",
+            "description": "A short but human-readable name of the room that the sensor is in."
+          },
+          "last_trigger_type" : {
+            "type" : "string",
+            "example" : "Occupied",
+            "description": "The last event the sensor recorded. Possible known values are Absent and Occupied, but this is not guaranteed. NB: only returned when return_states=true."
+          },
+          "survey_id" : {
+            "type" : "string",
+            "example" : "46",
+            "description": "The ID of the library/survey that was requested in the query parameter."
+          },
+          "room_type" : {
+            "type" : "string",
+            "example" : "Open Plan",
+            "description": "The type of the room."
+          },
+          "room_name" : {
+            "type" : "string",
+            "example" : "324",
+            "description": "The name of the room."
+          },
+          "room_id" : {
+            "type" : "string",
+            "example" : "291",
+            "description": "The ID of the room."
+          },
+          "location" : {
+            "type" : "string",
+            "example" : "",
+            "description": "The location of this sensor."
+          },
+          "survey_device_id" : {
+            "type" : "string",
+            "example" : "8420",
+            "description": "The ID of this survey device."
+          },
+          "share_id" : {
+            "type" : "string",
+            "example" : "None",
+            "description": "The Share ID."
+          },
+          "x_pos" : {
+            "type" : "string",
+            "example" : "32432.0",
+            "description": "The X coordinate position of the sensor on the map."
+          },
+          "description_1" : {
+            "type" : "string",
+            "example" : "Description 1."
+          },
+          "hardware_id" : {
+            "type" : "string",
+            "example" : "584001",
+            "description": "A unique ID for the sensor."
+          },
+          "pir_address" : {
+            "type" : "string",
+            "example" : "1",
+            "description": "PIR address."
+          },
+          "last_trigger_timestamp" : {
+            "type" : "string",
+            "example" : "2018-02-15T22:42:28+00:00",
```

Comment:
ooc: do we return time zone here? Our API seems to be really inconsistent 😅

Reply:
Yeah it look like we return exactly what comes from occupeye